_includes/manuals/1.16/4.1.3_roles_and_permissions.md

```diff
@@ -51,7 +51,7 @@ These determine the operations that are allowed to be performed upon the resourc
 
 When editing a filter there is a search field at the bottom that narrows the scope of the permissions granted to a subset of the resource objects. Most permission types support this search field however there are some exceptions. The permission for creating objects can't be limited by a search query because the object does not exist during creation. Therefore a user is granted the create permission if they are associated with any filter containing this permission (limited by search or not).
 
-Following table lists some of permissions and their impact:
```

Comment:
Shouldn't it still be `lists`, as it's just one table

Reply:
I see it as a single list of many options with different title, but exists for a single entity -> The table.
But I'm not sure regarding English grammar.

Reply:
"... table lists ..." is correct :)